core/src/main/scala/quasar/destination/gbq/GBQFlow.scala

```diff
@@ -246,11 +250,19 @@ final class GBQFlow[F[_]: Concurrent](
       Resource.eval(jobReq).flatMap(client.run).use { resp =>
         resp.status match {
           case Status.Ok => resp.headers match {
-            case Location(loc) => loc.uri.asRight[InitializationError[Json]].pure[F]
+            case Location(loc) => 
+              Sync[F].delay(log.info(s"Successfully initialised the job under URI '${loc.uri}'"))
```

Comment:
The only thing I'm unsure about is whether the URI is safe to log or not

Reply:
I don't have an example. I'll just log a simpler version to stay on the safe side.